LondonBritishLibrary/orient/BLorient787.xml

```diff
@@ -0,0 +1,288 @@
+                          </msItem>
+
+                          <msItem xml:id="ms_i1.11">
+                          <locus from="128r"></locus>
+                          <title type="complete" xml:lang="en">On His Descent into Hades</title>
+                          <textLang mainLang="gez"></textLang>
+                          </msItem>
+
+                          <msItem xml:id="ms_i1.12">
+                          <locus from="132r"></locus>
+                          <title type="complete" xml:lang="en">On the Resurrection of our Lord</title>
+                          <textLang mainLang="gez"></textLang>
+                          </msItem>
+
+                          <msItem xml:id="ms_i1.13">
+                          <locus from="145r"></locus>
+                          <title type="complete" xml:lang="en">On His Ascension</title>
+                          <textLang mainLang="gez"></textLang>
+                          </msItem>
+
+                          <msItem xml:id="ms_i1.14">
+                          <locus from="149r"></locus>
+                          <title type="complete" xml:lang="en">On His second Coming</title>
+                          <textLang mainLang="gez"></textLang>
+                          </msItem>
+
+                          <msItem xml:id="ms_i1.15">
+                          <locus from="153r"></locus>
+                          <title type="complete" xml:lang="en">On the Holy Spirit, the Paraclete</title>
+                          <textLang mainLang="gez"></textLang>
+                          </msItem>
+
+                          <msItem xml:id="ms_i1.16">
+                          <locus from="156v"></locus>
+                          <title type="complete" xml:lang="en">On the Equality (in Divinity) of the Holy Trinity</title>
+                          <textLang mainLang="gez"></textLang>
+                          </msItem>
+
+                          <colophon xml:lang="gez" xml:id="coloph1">
+                          <locus target="#164r"></locus>
+                          ተፈጸመ፡ መጽሐፈ፡ ሥጋዌ።
+                          </colophon>
+
+                        </msItem>
+
+
+
+                  </msContents>
+
+                        <physDesc>
+                          <objectDesc form="Codex">
+                              <supportDesc>
+                                  <support>
+                                      <material key="parchment"/>
+                                  </support>
+                                  <extent>
+                                      <measure unit="leaf">164</measure>
+                                      <measure unit="blank">1</measure>
+                                      <dimensions type="outer" unit="in">
+                                          <height>9.5</height>
+                                          <width>8.375</width>
+                                      </dimensions>
+                                  </extent>
```

Comment:
Right here you can add an element <foliation> with information on the unnumbered leaf.